React/CxxBridge/RCTRuntimeInitializeStateNotifier.h

```diff
@@ -0,0 +1,14 @@
+//
+//  RCTRuntimeInitializeStateNotifier.h
+//
+// TODO(OSS Candidate ISS#2710739)
+
+#import <AppKit/AppKit.h>
+
+#define RCTRuntimeInitializationEndNotificationName @"RCTRuntimeInitializationEndNotificationName"
```

Comment:
do we have to use macro? or just declare it as const NSString *

Reply:
I started with that but since the header is pulled into two different targets we get a duplicate symbol error building because it's defined in two targets